.github/workflows/verify.yml

```diff
@@ -0,0 +1,50 @@
+name: Verify
+
+on: [pull_request]
+
+concurrency:
+  group: verify-${{ github.head_ref }}
+  cancel-in-progress: true
+
+env:
+  NODE_VERSION: 16.x
+
+jobs:
+  build:
+    name: Build and Test
+    runs-on: ubuntu-latest
+    if: github.actor != 'dependabot[bot]'
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2.3.4
+
+      - name: Use Node.js ${{ env.NODE_VERSION }}
+        uses: actions/setup-node@v2.4.0
+        with:
+          node-version: ${{ env.NODE_VERSION }}
+          cache: yarn
+
+      - name: Configure package manager
+        run: |
+          echo Configuring NPM_READ_TOKEN globally for .npmrc
+          npm config set '//registry.npmjs.org/:_authToken' ${{ env.NPM_TOKEN }}
+          npm whoami
+        env:
+          NPM_TOKEN: ${{ secrets.NPM_READ_TOKEN }}
+
+      - name: Install dependencies
+        run: yarn install --frozen-lockfile
+
+      - name: Lint
+        run: yarn lint
+
+      - name: Check types
+        run: yarn types
+
+      - name: Test
+        run: yarn test --coverage --silent
```

Comment:
We should add a step to display the coverage results in a comment like here: https://github.com/reside-eng/next-bootstrap/blob/main/.github/workflows/verify.yml#L44-L48